docs/capabilities/system-open-source-health.md

```diff
@@ -37,9 +37,9 @@ If you change the category, the help mouseover will also change and provide you
 ## Open Source Health scans for 6 different risk areas
 
 “*Open Source Health*” scans for six different risk areas: 
-* **Known vulnerabilities:** publicly known and categorized vulnerabilities in a third-party dependency
-* **Freshness:** a measure of versioning compared to official/available versions. The count is the time difference with the date of the latest (publicly) available version. 
-* **Activity:** an estimation of (lack of) community activity as the time passed since the last release.
-* **Stability:** an estimate of stability by identifying seemingly unfinished versions such as alpha/beta/rc/0.x versions.
-* **Management:** possible risk originating from dependencies in binaries (e.g. JARs for Java) instead of configurations in package managers.
-* **Legal licenses:** third-party dependencies may have restricted licenses, that restrict how you can use it or e.g. obliges you to publish certain source code. This should always trigger a check by a legal expert. 
+<img src="../images/system-osh-icon-vulnerability.png" class="inline" /> **Known vulnerabilities:** publicly known and categorized vulnerabilities in a third-party dependency
+<img src="../images/system-osh-icon-freshness.png" class="inline" />  **Freshness:** a measure of versioning compared to official/available versions. The count is the time difference with the date of the latest (publicly) available version. 
+<img src="../images/system-osh-icon-activity.png" class="inline" />  **Activity:** an estimation of (lack of) community activity as the time passed since the last release.
```

Comment:
Maybe we can say: "an estimation of the lack of community activity since the last release."